doc/releases/changelog-dev.md

```diff
@@ -205,6 +205,9 @@ array([False, False])
 * The Pauli-X-term in `transmon_drive` has been removed in accordance with [1904.06560](https://arxiv.org/abs/1904.06560)
   [(#4418)](https://github.com/PennyLaneAI/pennylane/pull/4418/)
 
+* The gradients module no longer needs shot information passed to it explicitly, as the shots are on the tapes.
```

Comment:
Please could we also list this as a breaking change (removing shots from gradient transforms)

Reply:
it's been moved to the breaking changes section